src/run/handlers/request-context.cts

```diff
@@ -1,11 +1,18 @@
 import { AsyncLocalStorage } from 'node:async_hooks'
 
+import type { Context } from '@netlify/functions'
 import { LogLevel, systemLogger } from '@netlify/functions/internal'
 
 import type { NetlifyCachedRouteValue } from '../../shared/cache-types.cjs'
 
 type SystemLogger = typeof systemLogger
 
+// TODO: remove once https://github.com/netlify/serverless-functions-api/pull/219
```

Comment:
https://github.com/netlify/serverless-functions-api/pull/270

Reply:
While the PR link is outdated (I just moved this code from other file and didn't write this part now) - the types for `Context` still omit `waitUntil` so we still need to have `FutureContext` ( https://github.com/netlify/serverless-functions-api/blob/a6e469c32d472d55f5f2cce0e0683d147354df70/src/context.ts#L14 is what `Context` export from `@netlify/functions` is)

So I think we should have just:
```
// TODO: remove once public types are updated
```

Reply:
If we can find a link to include to the list of public types, that would also help

Reply:
Updated comment in https://github.com/opennextjs/opennextjs-netlify/pull/2717/commits/80de44f1ff0e32f0c79c5321d040e614a5e84107